qml/components-generic/VenueListItem.qml

```diff
@@ -83,12 +83,14 @@ ListItem {
     Label {
         id: closing
 
-        visible: !(model.open)
-                 //% "closed now"
-        text: qsTrId("id-venue-closed")
-        color: BVApp.Theme.disabledColor
-
-        width: !(model.open) ? contentWidth : 0
+        visible: !(model.closesSoon)
```

Comment:
With the below said, don't we simply want to show that if the restaurant is either closed (i.e. not open) or if it closes soon? Or am I missing sth.?

Reply:
Done.